sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/ReadBlobArgumentBinding.cs

```diff
@@ -39,5 +55,56 @@ public static async Task<Stream> TryBindStreamAsync(BlobBaseClient blob, Cancell
 
             return rawStream;
         }
+
+        public static async Task<ICacheAwareReadObject> TryBindCacheAwareAsync(BlobWithContainer<BlobBaseClient> blob, ValueBindingContext context, IFunctionDataCache functionDataCache)
+        {
+            try
+            {
+                // Generate the cache key for this blob
+                FunctionDataCacheKey cacheKey = await GetFunctionDataCacheKey(blob, context.CancellationToken).ConfigureAwait(false);
```

Comment:
Mention in the comment why we set to false the configure await.

Reply:
I'd not do this. There's analyzer plugged into build that makes sure every `awat Foo().ConfigureAwait(false)` does this.